pygerm/handler.py

```diff
@@ -22,28 +23,61 @@ def close(self):
 class BinaryGeRMHandler(HandlerBase):
     specs = {'BinaryGeRM'}
 
-    def __init__(self, fpath):
-        # TODO : don't save the raw data (here for debugging)
-        raw_data = np.fromfile(fpath, dtype='>u4')
-        # TODO : when simulated data comes in, verify this is correct
-        # endianness and correct for it, don't just raise error
+    def __init__(self, fpath, chunksize=None):
+        ''' Binary GeRM handler.
+
+            Parameters
+            ----------
+            chunksize : int, optional
+                if specified, this turns result into a dask array
+                This array is a lazy loaded array. To obtain the results
+                one must call the .compute() method.
+                This is useful when the data is too large to fit in memory.
+
+            Notes
+            -----
+            When using chunksize, use functools.partial, and functools.wraps
+            Example:
```

Comment:
Writing this as 

```rst

Example ::

   from functools import partial, wraps

```

will make the code block render as code when built with sphinx